subprojects/testing-native/src/main/java/org/gradle/nativeplatform/test/xctest/tasks/InstallXCTestBundle.java

```diff
@@ -160,6 +162,7 @@ public RegularFileProperty getBundleBinaryFile() {
     @SkipWhenEmpty
     @Nullable
     @Optional
+    @PathSensitive(PathSensitivity.RELATIVE)
```

Comment:
Single input file should not be `RELATIVE`.

Reply:
Changed to `NAME_ONLY`.